src/tests/ftest/util/daos_utils.py

```diff
@@ -495,6 +495,8 @@ def container_list_attrs(self, pool, cont, sys_name=None):
         # aa bb
         # attr48
         match = re.findall(r"\n([\S ]+)", self.result.stdout_text)
+        # strip all trailing spaces from new daos cmd output formatting
+	match = [x.rstrip() for x in match]
```

Comment:
(pylint-syntax-error) inconsistent use of tabs and spaces in indentation (<unknown>, line 499)